.github/workflows/pre-merge-ci.yaml

```diff
@@ -24,3 +24,17 @@ jobs:
 
     - name: Run checks
       run: make ci
+
+    - name: Install gomplate
+      run: make install-gomplate GOMPLATE_BIN=$HOME/.local/bin
+
+    - name: Build docs
+      run: make docs
+
+    # FIXME: This produces a zip file so it's not much use.
```

Comment:
GitHub pages I think is the way to go. This is what I did for https://lcarva.github.io/review-rot/

Relevant commits:
* https://github.com/lcarva/review-rot/commit/56702d02af1e07c5c272b57c8b017b89bb043c70
* https://github.com/lcarva/review-rot/commit/c03948b019afdd6b052f7286645faf350a6d346c
* https://github.com/lcarva/review-rot/commit/0a09019bf3895d1fe770759e11af249d7cd500e8

Reply:
Looks like it's working, see https://simonbaird.github.io/ec-policies/ .

Actually didn't need to do much, other than add a docs/_config.yml file turn it on in settings.